tools/buildsteps/defaultenv

```diff
@@ -4,7 +4,7 @@ NDK_VERSION=${NDK_VERSION:-"Default"}
 NDK_API=${NDK_API:-"Default"}
 Configuration=${Configuration:-"Default"}
 XBMC_DEPENDS_ROOT=${XBMC_DEPENDS_ROOT:-"Default"}
-DARWIN_ARM_CPU=${DARWIN_ARM_CPU:-"Default"}
+DARWIN_ARM_CPU=${DARWIN_ARM_CPU:-"arm64"}
```

Comment:
Is there any case where the darwin arm cpu isn't arm64?

Reply:
Not currently

Reply:
there exists `arm64e`, but it can run `arm64` code

Reply:
I understand something should be changed because it's the only option I just don't know what you would like changed.

Reply:
Replace the usage of `DARWIN_ARM_CPU` with it's actual value.